views/pulsar/v2/helpers/form.html.twig

```diff
@@ -1282,7 +1358,7 @@ value  | string | The string or html markup to render inside the help block
 {% spaceless %}
 
     {% if options.value is defined and options.value is not empty %}
-        <span class="help-block">
+        <span class="help-block"{% if options.id is defined and options.id is not empty %} id="{{- options.id|raw -}}"{% endif %}>
```

Comment:
Why is `raw` required here? This will be a problem due to opening up a potential XSS vulnerability.